tests/zfs-tests/tests/functional/cli_root/zfs_mount/zfs_mount_013_pos.ksh

```diff
@@ -57,6 +57,14 @@ log_must eval ZFS_MOUNT_HELPER=1 zfs $mountcmd -v $TESTPOOL
 log_must mounted $TESTPOOL
 force_unmount $TESTPOOL
 
+log_note "Verify CWD prefix filter <dataset> <path>"
+cd / && zfs set mountpoint=legacy $TESTPOOL && mkdir -p $mntpoint || log_fail
```

Comment:
It might be nice to rewrite this like so if one of these steps happens to fails it's obvious which one.  

```
log_must cd /
log_must zfs set mountpoint=legacy $TESTPOOL
log_must mkdir -p $mntpoint
```